src/Spectre.libClassifier.Tests/SvmTest.cpp

```diff
@@ -71,6 +72,28 @@ TEST_F(SvmTest, predicts_sign)
     }
 }
 
+TEST_F(SvmTest, predicts_small_data)
+{
+    const std::vector<Label> tempLabels = { 1 };
+    const std::vector<DataType> tempFeatures = {
+        1., 1.
+    };
+    OpenCvDataset tempData = std::move(OpenCvDataset(tempFeatures, tempLabels));
```

Comment:
We are moving created instance to the same place it is created?

Reply:
Nice catch, weird line, done